fluency-fluentd/src/test/java/org/komamitsu/fluency/fluentd/FluencyBuilderForFluentdTest.java

```diff
@@ -17,6 +17,7 @@
 package org.komamitsu.fluency.fluentd;
 
 import org.junit.Test;
+import org.junit.runner.RunWith;
```

Comment:
Is this import needed?

Reply:
oops sorry this was left off from my running the junit 5... removed now